scikitplot/decomposition.py

```diff
@@ -160,6 +170,20 @@ def plot_pca_2d_projection(clf, X, y, title='PCA 2-D Projection', ax=None,
     for label, color in zip(classes, colors):
         ax.scatter(transformed_X[y == label, 0], transformed_X[y == label, 1],
                    alpha=0.8, lw=2, label=label, color=color)
+
+    if biplot:
+        vectors = np.transpose(clf.components_[:2, :])
+        xs = transformed_X[:, 0]
+        ys = transformed_X[:, 1]
+        for i in range(vectors.shape[0]):
+            ax.annotate("", xy=(vectors[i, 0]*max(xs), vectors[i, 1] * max(ys)),
```

Comment:
Yea, I can work on a test.

You are absolutely correct, I should have considered matrix multiplication. 
```vectors_scaled``` seems like an appropriate enough name. Its simply a multiplier to scale the biplot vectors so that they may be better visualized in the pca plane.